### PR TITLE
Title page and about pages

### PR DESCRIPTION
**Pull request by waltmao98**
There are some minor details that could be improved, but all of the functionality is there. @JerryXie98 you can use the Next and Previous buttons.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,13 @@
   "dependencies": {
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1"
+    "react-redux": "^7.1.0",
```
Comment (suzyng83209) [inline package.json]: we're not using redux anywhere, please remove all unnecessary dependencies including `react-router-dom` and whatever else

File: src/App.css [hunk 2]
```diff
@@ -44,4 +44,4 @@ h1 {
 
   box-shadow: 1px 1px 8px rgba(0, 0, 0, 0.07);
   border-radius: 8px;
-}
+}
```
Comment (suzyng83209) [inline src/App.css]: nit: new line

File: src/actions/answerActions.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import {
```
Comment (suzyng83209) [inline src/actions/answerActions.js]: why is this file even here.

File: src/components/AboutThisTool.jsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Footer from "./Footer";
+import { Link } from "react-router-dom";
+
+import '../App.css';
+import QuestionnaireHeader from "./Header";
+
+const AboutThisTool = ({ resetData }) => {
```
Comment (suzyng83209) [inline src/components/AboutThisTool.jsx]: you don't have `resetData` anymore. Remove it.

File: src/components/AboutThisTool.jsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            </button>
+				</div>
+        </div>
+      </div>
+      <Footer />
+    </div>
+  );
+};
+
+const ButtonGroup = (resetData) => {
```
Comment (suzyng83209) [inline src/components/AboutThisTool.jsx]: same here.

File: src/components/BottomNavigationButtons.jsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+        <button className="secondary bottom-button" onClick={() => decrement()}>
+            Previous
+        </button>
+        <button className="primary bottom-button next-button" onClick={() => increment()}>
+            Next
+        </button>
+    </div>);
+}
+
+export default BottomNavigationButtons;
```
Comment (suzyng83209) [inline src/components/BottomNavigationButtons.jsx]: nit: new line

File: src/components/ProgressIndicator.jsx [hunk 7]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const sections = [0,1,2]
```
Comment (suzyng83209) [inline src/components/ProgressIndicator.jsx]: common practice: capitalize constant local variables.

File: src/containers/AboutThisToolContainer.jsx [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
```
Comment (suzyng83209) [inline src/containers/AboutThisToolContainer.jsx]: NOPE. no redux.

File: src/containers/TitlePageContainer.jsx [hunk 9]
```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
```
Comment (suzyng83209) [inline src/containers/TitlePageContainer.jsx]: Same here. You shouldn't be seeing any `___Container` files. No containers

File: src/index.js [hunk 10]
```diff
@@ -1,12 +1,37 @@
...
+    reducers,
+    initialStore,
+    composeEnhancers(applyMiddleware(...middleware))
+);
+
+storage.createLoader(engine)(store);
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+    <Provider store={store}>
```
Comment (suzyng83209) [inline src/index.js]: again remove all redux from the app.

File: src/initialStore.js [hunk 11]
```diff
@@ -0,0 +1,6 @@
+const initialStore = {
```
Comment (suzyng83209) [inline src/initialStore.js]: ditto. remove.

File: src/reducers/index.js [hunk 12]
```diff
@@ -0,0 +1,11 @@
+import { combineReducers } from "redux";
```
Comment (suzyng83209) [inline src/reducers/index.js]: `reducers` should not be a folder in this app.

**Comment by waltmao98** [comment]
Sorry about the confusion with the Redux stuff. When I worked on the old framework before noticing your changes on the new framework I worked with the Redux stuff. Those changes seemed to have been included in this change. I've manually went through the diff and removed all old framework changes.

**Pull request merged**